python/ray/dataframe/test/test_dataframe.py

```diff
@@ -1317,12 +1319,33 @@ def test_dropna_inplace(ray_df, pd_df):
     ray_df.dropna(thresh=2, inplace=True)
     pd_df.dropna(thresh=2, inplace=True)
 
-    ray_df_equals_pandas(ray_df, pd_df)
+    assert ray_df_equals_pandas(ray_df, pd_df)
 
     ray_df.dropna(axis=1, how='any', inplace=True)
     pd_df.dropna(axis=1, how='any', inplace=True)
 
-    ray_df_equals_pandas(ray_df, pd_df)
+    assert ray_df_equals_pandas(ray_df, pd_df)
+
+
+@pytest.fixture
+def test_dropna_multiple_axes(ray_df, pd_df):
+    ray_df = ray_df.copy()
+    pd_df = pd_df.copy()
+    cp = ray_df.copy()
+    result = ray_df.dropna(how='all', axis=[0, 1])
+    result2 = ray_df.dropna(how='all', axis=(0, 1))
+    expected = pd_df.dropna(how='all').dropna(how='all', axis=1)
+
+    assert ray_df_equals_pandas(result, expected)
+    assert ray_df_equals_pandas(result2, expected)
+
+    assert ray_df_equals_pandas(result, expected)
+    assert ray_df_equals_pandas(result2, expected)
+    assert ray_df_equals(ray_df, cp)
+
+    inp = ray_df.copy()
```

Comment:
You already make a copy above, use it since none of those operations mutated the copy.